auth0/src/main/java/com/auth0/android/request/DefaultClient.kt

```diff
@@ -75,16 +84,16 @@ public class DefaultClient() : NetworkingClient {
         }
 
         // timeouts
-        builder.connectTimeout(connectTimeout, TimeUnit.SECONDS)
-        builder.readTimeout(readTimeout, TimeUnit.SECONDS)
+        builder.connectTimeout(connectTimeout.toLong(), TimeUnit.SECONDS)
+        builder.readTimeout(readTimeout.toLong(), TimeUnit.SECONDS)
 
-        client = builder.build()
+        okHttpClient = builder.build()
     }
 
 
-    private companion object {
-        private const val DEFAULT_TIMEOUT_SECONDS: Long = 10
-        private val APPLICATION_JSON_UTF8: MediaType =
+    internal companion object {
+        const val DEFAULT_TIMEOUT_SECONDS: Int = 10
```

Comment:
I changed this to an Int, to enable using it in `Auth0` where the corresponding values are Int's. As the setting in OkHttp should not exceed `Integer.MAX_VALUE`, I think having it as an int makes sense anyways.